README.md

```diff
@@ -15,13 +15,94 @@ refer to the documentation below for how to do that.
 ### Build requirements
  - a C++11 compiler (clang/gcc 4.7)
  - gnumake
- - cmake
- - boost
+ - CMake
+ - Boost
  - ruby (2.0 and newer)
- - [leatherman][leatherman](0.3.5 or newer)
+ - [leatherman][leatherman] (0.3.5 or newer)
  - [cpp-pcp-client][cpp-pcp-client] (master)
 
-Build with make and make install
+Initial Setup
```

Comment:
Perhaps you can get rid of the above ## "Building from source" section and rename the above ### "Build requirements" one to ### "Dependencies", since we have a new "Build" section below.